appveyor.yml

```diff
@@ -36,7 +36,7 @@ cache:
 pull_requests:
   do_not_increment_build_number: true
 
-skip_branch_with_pr: true
+#skip_branch_with_pr: true
```

Comment:
Leave this here, it helps avoid building twice for every commit on pull requests for feature branches